modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Enabling or disabling checkboxes did not set the correct classes and attributes
+//#TINY-4189
+
+In previous version of {productname}, the checkbox element’s disabled state was not correctly applied to all elements. As a result, the state was neither properly removed nor added as intended.
```

Comment:
It could be worth mentioning that it'd leave the checkboxes and such in a state that suggested they were enabled/disabled ( whichever they started as ) despite not being so. Seeminly disabled checkboxes could still be checked if the user clicked them, and the other way around.
Not a requirement though, this is enough too.

Reply:
Edited to include this suggested addition.